src/Livewire/Auth/Login.php

```diff
@@ -47,7 +46,14 @@ public function login(): bool
         $target = Session::get('url.intended', route($this->dashboardRoute));
         $this->validate();
 
-        $login = $this->tryLogin();
+        if ($this->password) {
+            $login = $this->tryLogin();
+        } else {
+            $this->sendMagicLink();
+            $this->notification()->success(__('Login link sent, check your inbox'));
```

Comment:
Why show success notification even if the email was not sent?

Reply:
Prevents brute force